core/src/test/java/io/questdb/test/griffin/ParquetTest.java

```diff
@@ -275,4 +270,94 @@ public void testSinglePartition() throws Exception {
             );
         });
     }
+
+    @Test
+    public void testSymbols() throws Exception {
```

Comment:
Would be great to add some tests around column tops, so that we cover the corresponding code in `column_chunk_to_pages`.

Reply:
Tracked as part of testing section of parent PR.